cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -175,6 +175,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodeList() {
        ++trajectory_id) {
     const auto& single_trajectory_nodes = all_trajectory_nodes[trajectory_id];
     visualization_msgs::Marker marker;
+    marker.ns = "Trajectory " + std::to_string(trajectory_id);
     marker.id = marker_id++;
```

Comment:
Sorry, I haven't read this carefully. Perhaps we could. You want to make `marker` a local variable of one scope above and to keep the marker id inside of it?